sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_aws_sdk.ipynb

```diff
@@ -0,0 +1,889 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Using retrieve API of Amazon SageMaker Python SDK for demo purposes,\n",
+    "# refer list of image URIs to if you wish to skip using this API.\n",
+    "clarify_image_uri = retrieve(region=region, framework=\"clarify\", version=\"1.0\")\n",
+    "print(f\"Clarify Image URI: {clarify_image_uri}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Detecting Bias\n",
+    "SageMaker Clarify helps you detect possible pre- and post-training biases using a variety of metrics."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Configure a SageMaker Clarify Processing Job Container's Input and Output Parameters \n",
+    "The Processing Job requires that you specify the following input parameters: a dataset files with input name \"dataset\" as Amazon S3 object or prefix, and an analysis configuration file with input name \"analysis_config\" as an Amazon S3 object. The job also requires an output parameter: the output location as an Amazon S3 prefix."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "bias_analysis_config_path = \"s3://{}/{}/bias_analysis_config.json\".format(bucket, prefix)\n",
+    "analysis_result_path = \"s3://{}/{}/output\".format(bucket, prefix)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Configure Analysis Config\n",
+    "The inputs for the analysis are configured by the parameters of the ProcessingInput API. The \"analysis_config\" value of the input_name specifies the JSON file that contains the configuration values. The path to the JSON file is provided in the source parameter of ProcessingInput. More details on configuring an analysis config can be referred [here](https://docs.aws.amazon.com/sagemaker/latest/dg/clarify-processing-job-configure-analysis.html).\n",
+    "\n",
+    "For our example use case we will be using the following analysis config:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!echo\n",
+    "!cat bias_analysis_config.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`bias_analysis_config.json` here contains configuration values for detecting bias using a clarify container:\n",
+    "* `dataset_type` specify the format of your dataset, for thsi example as we are using csv dataset this will be `text/csv`\n",
+    "* `headers` is the list of column names in the dataset\n",
+    "* `label` specified the target attribute for the model to be used for bias metrics. Specified as a column name, or as index if the dataset format is CSV\n",
```

Comment:
General: I think the description here should focus on what/why we chose for this example. E.g. 

* `label` specifies the ground truth label, which is also known as observed label or target attribute. It is used for many bias metrics. In this example, the "Target" column has the ground truth label.

No need to mention index because the index doesn't use index to locate columns.